examples/add_items_to_index.py

```diff
@@ -0,0 +1,42 @@
+"""
+This example shows how to add your items in autocomplete index
```

Comment:
"In *an* autocomplete index"? 

also do we still want to call this an `autocomplete` index? I think since we're starting to support search, we want to move away from using the word `autocomplete` in places that are not specifically related to the autocomplete dropdown. so this could even be "how to add items to your index".